js/package.json

```diff
@@ -40,11 +40,18 @@
     "@jupyterlab/application": "^3.0.0",
     "@jupyterlab/apputils": "^3.0.0",
     "@jupyterlab/coreutils": "^5.0.0",
+    "@jupyterlab/launcher": "^3.0.2",
     "@lumino/disposable": "^1.1.2",
-    "requests-helper": "^0.1.5"
+    "@types/node": "^14.14.22",
+    "axios": "^0.21.1",
+    "image-pixels": "^2.2.2",
+    "imagetracerjs": "^1.2.6",
```

Comment:
remove dep

Reply:
needed to convert png/jpg icons to svg's jupyterlab can display in launcher

Reply:
see above